Assets.md

```diff
@@ -1,6 +1,9 @@
 # Assets
-### When do we use them?
-We use asset catalogs to manage our application's assets.
+### How do we use them?
+We use asset catalogs to organize our application’s images and colors.
```

Comment:
Any other of the supported media types of asset catalogs we should consider?

Reply:
Good question. I only want to apply specific rules to images and colors at this time as I have in the "Key Considerations", so I’m going to revert "images and colors" to "assets". I don’t want to get into the specifics of how we might interact with the many many things that can go into asset catalogs, beyond the fact that they should exist there, which the original statement covers.

Reply:
Makes sense to me.